include/gunrock/algorithms/async/async_enactor.hxx

```diff
@@ -0,0 +1,68 @@
+
+#pragma once
+
+#include <gunrock/algorithms/async/queue.cuh>
+
+namespace gunrock {
+  
+template <typename algorithm_problem_t>
+struct async_enactor_t {
+
```

Comment:
For now, I think it makes sense to keep experimental async functionality in it's own enactor.  Interested in your thoughts on that though.